### ISSUE TITLE
Synth failed, but it succeeded.

### ISSUE THREAD

**Issue by mlandrum22** [EdenServer/community#2665]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

I used a lightning crystal with an emperor fish, looked away and my log said "Synthesis failed. You lost the crystal you were using." So I hit my /lastsynth macro and it said "Synthesis cancelled."

Checked my inventory and I had 4 Emperor Roes (NQ version of the synth). 

I wasn't watching, so I'm not sure what animation it gave.

**Comment by Mozzy4Ever** [comment]
"Synthesis cancelled" comes up when you don't have the materials for the synth you're trying to make. You sure you have enough lightning crystals?

**Comment by mlandrum22** [comment]
Yes, and like I said, I had the mats of a successful synth afterwards.
